combinations/hash.tsv

```diff
@@ -249,3 +249,4 @@ pymuonsuite=0.2.1
 muspinsim=1.1.0
 changeo=1.2.0,igblast=1.17.1,wget=1.20.1	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	0
 cnvkit=0.9.9,samtools=1.15.1
+deeptools=3.5.1,samtools=1.15.1	bgruening/busybox-bash:0.1	0
```

Comment:
`bgruening/busybox-bash:0.1` has issues with resolving DNS on singularity (https://github.com/bioconda/bioconda-recipes/issues/11583). Any reason to not use the new default image (`quay.io/bioconda/base-glibc-busybox-bash`, or leaving the field empty, that has the same effect ) ?